src/modules/profile/components/profile.styles.scss

```diff
@@ -0,0 +1,285 @@
+// Variables
```

Comment:
I like the use of sass here; variables and all. Do you think this stylesheet could be made simpler? I think it is overcomplicated. Also, having all the styles of a page in a single CSS file tends to be cumbersome and difficult to maintain. I would propose every component to have its own stylesheet imported to the CSS files would be kept as short and readable as possible.

Reply:
Done. If you have any other suggestions on this topic please let me know.